src/app/main/component/shared/components/tags-select/tags-select.component.ts

```diff
@@ -31,4 +35,8 @@ export class TagsSelectComponent {
   getLangValue(valUa: string, valEn: string): string {
     return this.langService.getLangValue(valUa, valEn) as string;
   }
+
+  changeValueIsActive(): void {
+    this.tagsList = this.tagsList ? this.tagsList.map((tags) => ({ ...tags, isActive: false })) : this.tagsList;
```

Comment:
is there any sense in creating a new function for this if we call it only ones in `ngOnInit`?)